### DISCUSSION TITLE
Different hit scores for early and late hits

### DISCUSSION THREAD

**Discussion by damga123** [ppy/osu#14038] [Ideas] [answered]
Similar to what can be sean in A Dance of Fire and Ice, it would be nice if there were different hit scores for early and late hits.  These could of course be customized. This would also open up the option to bring back slider end dropping hit scores, because as far is I know there is no visible hit scoring for dropping a slider end right now.

**Comment by peppy** [comment, marked as answer]
We do plan on this, but it will likely not come until we get around to updating the judgement graphics in the default skin.
